src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -333,7 +368,7 @@ impl NodeModel {
         self.main_area.mod_position(|t| t.x = width/2.0);
         self.drag_area.mod_position(|t| t.x = width/2.0);
 
-        let action_bar_width = 200.0;
+        let action_bar_width = 180.0;
```

Comment:
Constant in code. (Probably this was me, but would probably still be better to have it based on the node width)

Reply:
This is not about node width. But yeah, its strange. Ill put it into constants for now